src/applications/coronavirus-vaccination/hooks/useSubmitForm.jsx

```diff
@@ -2,26 +2,34 @@ import { useState, useCallback } from 'react';
 import { requestStates } from 'platform/utilities/constants';
 
 import { saveForm } from '../api';
+import useRecaptcha from './useRecaptcha';
 
 export default function useSubmitForm() {
   const [status, setSubmitStatus] = useState(requestStates.notCalled);
+  const [executeRecaptcha] = useRecaptcha();
+  const appActionID = 'coronavirus_vaccine_submit';
 
   const submit = useCallback(formData => {
-    async function sendToApi() {
+    async function sendToApi(token) {
       const normalized = {
         ...formData,
         zipCodeDetails: formData.locationDetails,
+        recaptchaToken: formData.isIdentityVerified ? null : token,
       };
+
       try {
         await saveForm(normalized);
         setSubmitStatus(requestStates.succeeded);
       } catch (error) {
         setSubmitStatus(requestStates.failed);
       }
     }
-
+    if (!formData.isIdentityVerified) {
+      executeRecaptcha(sendToApi, appActionID);
+    } else {
+      sendToApi();
+    }
```

Comment:
Do we need to feature flag this (via Flipper or a check on the environment) or anything until the API portion is ready?

Reply:
almost certainly!